src/main/java/com/erigitic/jobs/TEJobManager.java

```diff
@@ -677,41 +682,72 @@ public void onPlayerBlockBreak(ChangeBlockEvent.Break event) {
             String playerJob = getPlayerJob(player);
             Optional<TEJob> optPlayerJob = getJob(playerJob, true);
 
-            String blockName = event.getTransactions().get(0).getOriginal().getState().getType().getName();
+            BlockState state = event.getTransactions().get(0).getOriginal().getState();
+            String blockName = state.getType().getName();
             Optional<UUID> blockCreator = event.getTransactions().get(0).getOriginal().getCreator();
 
             if (optPlayerJob.isPresent()) {
-                // Prevent blocks placed by other players from counting towards a job
-                if (!blockCreator.isPresent()) {
-                    Optional<TEActionReward> reward = Optional.empty();
-                    List<String> sets = optPlayerJob.get().getSets();
-
-                    for (String s : sets) {
-                        Optional<TEJobSet> optSet = getJobSet(s);
-                        if (!optSet.isPresent()) {
-                            logger.warn("[TE] Job " + getPlayerJob(player) + " has the nonexistent set \"" + s + "\"");
-
-                            continue;
-                        }
+                Optional<TEActionReward> reward = Optional.empty();
+                List<String> sets = optPlayerJob.get().getSets();
 
-                        reward = optSet.get().getRewardFor("break", blockName);
+                for (String s : sets) {
+                    Optional<TEJobSet> optSet = getJobSet(s);
+                    if (!optSet.isPresent()) {
+                        logger.warn("Job " + playerJob + " has the nonexistent set \"" + s + "\"");
+                        continue;
                     }
+                    Optional<TEActionReward> cr = optSet.get().getRewardFor("break", blockName);
```

Comment:
Change from `cr` to something more descriptive such as `currentReward` or to whatever you think fits best.